test/helper.js

```diff
@@ -51,15 +51,15 @@ mocha.before(async () => {
     helper.Client = overwrites['Client'] = data.Client.setup({
       table: 'Client',
       account: 'inMemory',
+      credentials: null,
       cryptoKey,
       signingKey,
-      context: {resolver: overwrites.resolver},
```

Comment:
This appears to be passed in to the functions that need it now (just noting for my reference..)